src/sentry/sentry_metrics/querying/api.py

```diff
@@ -133,16 +159,46 @@ def group_bys(self) -> Sequence[str]:
         return group_bys
 
     @property
-    def data(self) -> Mapping[str, Any]:
-        return self.result["data"]
+    def series(self) -> Sequence[Mapping[str, Any]]:
+        return self.result["series"]["data"]
 
     @property
-    def totals(self) -> Mapping[str, Any]:
-        return self.result["totals"]
+    def totals(self) -> Sequence[Mapping[str, Any]]:
+        return self.result["totals"]["data"]
 
     @property
     def meta(self) -> Sequence[Mapping[str, str]]:
-        return self.result["meta"]
+        # We assume that we always have totals.
```

Comment:
nit: can we make this assumption? Isn't this controlled by the API param `includeTotals` and then `with_totals`?

Reply:
This was copied from another PR of mine which introduced this new querying system. The assumption there held since it was always true that we had totals and in this PR the same holds, it's controlled in the `schedule`. However, I can definitely fallback to series metadata in case the totals one is not found.